src/app/static/src/components/downloadResults/DownloadResults.vue

```diff
@@ -187,9 +187,24 @@
             }
         },
         methods: {
+            downloadUrl(downloadId: string) {
+                return `/download/result/${downloadId}`;
+            },
             handleUploadModal() {
                 this.uploadModalOpen = true;
             },
+            handleDownloadResult(downloadResults: DownloadResultsDependency) {
+                window.location.assign(this.downloadUrl(downloadResults.downloadId));
+            },
+            downloadSpectrumOutput() {
```

Comment:
Confused as to what's going on here - these are duplicating methods mapped below via `mapActionByName`